imap/mbpath.c

```diff
@@ -448,6 +448,8 @@ int main(int argc, char **argv)
         fatal(error_message(r), EX_SOFTWARE);
     }
 
+    mbpath_namespace.isutf8 = opts.utf8;
+
```

Comment:
Could `mboxname_init_namespace()` do this?

Reply:
@elliefm  I thought about that.  I think I prefer making the isadmin argument a bitmask of flags rather than adding a third argument.  Your thoughts?

Reply:
No strong opinion either way, and I'll accept whatever you choose, but some off the cuff thoughts:

* adding a bitmask is a (small) bunch of extra infrastructure, which is more work than just adding one argument
* but bitmask fields, which have names, are easier to read than a list of ones and zeroes (though we usually comment those, like `foo(..., /*isadmin*/ 0, ...)`, which solves that issue)
* the `mboxname_init_namespace()` function signature has only changed once in as long as it's existed (`isadmin` was originally called `force_std` -- which now that I see that, explains all the "force standard" comments everywhere, hah). I'm not sure we expect it to change much in the future after whatever we do here
* the infrastructure cost of a bitmask makes future changes easier, but if we don't expect any...

Myself, I would probably just add the extra argument, and not worry about  converting to a bitmask unless we started needing a third flag someday.  But that's just me.

Reply:
@elliefm Implemented bitmask